github-actions/post-approval-changes/lib/main.ts

```diff
@@ -57,6 +59,10 @@ async function runPostApprovalChangesAction(client: Octokit): Promise<void> {
     return;
   }
 
+  if (googleOwnedRobots.includes(pr.user.login)) {
+    core.info('PR author is a robot owned by Google. Post approval changes are allowed.');
```

Comment:
Did you mean to return here or something?

Reply:
No, just logging. Joke, you're right. good catch 🤦 too many drive-by things today!